.github/composer-config.json

```diff
@@ -4,6 +4,7 @@
         "preferred-install": {
             "symfony/form": "source",
             "symfony/http-kernel": "source",
+            "symfony/mailer": "source",
```

Comment:
Is this correct and needed?

Reply:
it is needed yes, to install the Tests/ dir in vendors when testing transports